File: samples/wordnik-api-java/com/wordnik/client/model/Note.java
package com.wordnik.client.model;

import java.util.*;
public class Note {
  private String noteType = null;
  private List<String> appliesTo = new ArrayList<String>();
  private String value = null;
  private int pos = 0;
  public String getNoteType() {
    return noteType;
  }
  public void setNoteType(String noteType) {
    this.noteType = noteType;
  }

  public List<String> getAppliesTo() {
    return appliesTo;
  }
  public void setAppliesTo(List<String> appliesTo) {
    this.appliesTo = appliesTo;
  }

  public String getValue() {
    return value;
  }
  public void setValue(String value) {
    this.value = value;
  }

  public int getPos() {
    return pos;
  }
  public void setPos(int pos) {
    this.pos = pos;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Note {\n");
    sb.append("  noteType: ").append(noteType).append("\n");
    sb.append("  appliesTo: ").append(appliesTo).append("\n");
    sb.append("  value: ").append(value).append("\n");
    sb.append("  pos: ").append(pos).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: samples/wordnik-api-scala/src/main/scala/com/wordnik/client/model/Note.scala
package com.wordnik.client.model

import scala.reflect.BeanProperty

class Note {
  @BeanProperty var noteType: String = _
  @BeanProperty var appliesTo: java.util.List[String] = new java.util.ArrayList[String]()
  @BeanProperty var value: String = _
  @BeanProperty var pos: Int = 0
  override def toString: String = {
    val sb = new StringBuilder
    sb.append("class Note {\n")
    sb.append("  noteType: ").append(noteType).append("\n")
    sb.append("  appliesTo: ").append(appliesTo).append("\n")
    sb.append("  value: ").append(value).append("\n")
    sb.append("  pos: ").append(pos).append("\n")
    sb.append("}\n")
    sb.toString
  }
}
